pkg/applicationprofilemanager/v1/applicationprofile_manager.go

```diff
@@ -243,7 +246,7 @@ func (am *ApplicationProfileManager) saveProfile(ctx context.Context, watchedCon
 	// sleep for container index second to desynchronize the profiles saving
 	time.Sleep(time.Duration(watchedContainer.ContainerIndex) * time.Second)
 
-	if droppedEvents := am.droppedEvents.Get(watchedContainer.K8sContainerID); droppedEvents {
+	if am.droppedEventsContainers.ContainsOne(watchedContainer.K8sContainerID) {
```

Comment:
Searching in a map is faster why using a slice?

Reply:
This is a set not a slice